packages/regenesis-surgery/scripts/data.ts

```diff
@@ -89,7 +89,7 @@ const getChainId = async (
   provider: ethers.providers.JsonRpcProvider
 ): Promise<number> => {
   const response = await provider.send('eth_chainId', [])
-  return ethers.BigNumber.from(response.result).toNumber()
+  return ethers.BigNumber.from(response).toNumber()
```

Comment:
cc @tynes I had to make this change for the script to work. What node did you test against where you got a `result` field in the object that you got back?

Reply:
I was testing locally against one of our cloud nodes, sorry about that

Reply:
Which cloud node? If it's a possibility to get an object with a `result` field back, we should handle that case.

Reply:
`call` will return the `result` field, see https://github.com/ethers-io/ethers.js/blob/b1458989761c11bf626591706aa4ce98dae2d6a9/packages/providers/src.ts/json-rpc-provider.ts#L103